app/accuracy_app/accuracy_check.cpp

```diff
@@ -0,0 +1,23 @@
+#include <opencv2/opencv.hpp>
+#include <iostream>
+#include <filesystem>
+
+int main() {
+  std::string directory = "./photos";
+
+  for (const auto& entry : std::filesystem::directory_iterator(directory)) {
```

Comment:
```
namespace fs = std::filesystem;
```
I suggest using this, it is a common widely-used way to simplify std::filesystem related code